test/scraping/cycling/procyclingstats/raceStartLists.test.js

```
@@ -0,0 +1,33 @@
+  {
+    race: "Tour Down Under",
+    input:
+      "test/scraping/cycling/procyclingstats/html/raceStartlist-2025-tour-down-under.html",
+    output:
+      "test/scraping/cycling/procyclingstats/fixtures/raceStartlist-2025-tour-down-under.json",
+  },
+])(`$filterYear $filterClass races`, (data) => {
+  let html, expectedResults;
+
+  beforeAll(async () => {
+    const input = Bun.file(data.input);
```

ESLint

[error] 'Bun' is not defined.

(no-undef)